src/components/BottomSheet/components/BottomSheetContainer.tsx

```diff
@@ -0,0 +1,44 @@
+import React, {memo, useMemo} from 'react'
+import type {ViewProps} from 'react-native'
+import {StyleProp, StyleSheet, ViewStyle} from 'react-native'
+import {GestureHandlerRootView} from 'react-native-gesture-handler'
+import Animated, {useAnimatedProps} from 'react-native-reanimated'
+import {useBottomSheet} from '../hooks/useBottomSheet'
```

Comment:
After adding useBottomSheet to index.ts, please shorten this import

Reply:
updated